src/components/contact/IFXContactCreateEdit.vue

```diff
@@ -1,31 +1,75 @@
 <script>
 import { mapActions } from 'vuex'
-import IFXItemCreateEditMixin from '@/components/item/IFXItemCreateEditMixin'
-import IFXContactMixin from '@/components/contact/IFXContactMixin'
+import IFXEmailContactCreateEdit from '@/components/contact/IFXEmailContactCreateEdit'
+import IFXPhoneContactCreateEdit from '@/components/contact/IFXPhoneContactCreateEdit'
+import IFXFullContactCreateEdit from '@/components/contact/IFXFullContactCreateEdit'
 
 export default {
   name: 'IFXContactCreateEdit',
-  mixins: [IFXContactMixin, IFXItemCreateEditMixin],
+  components: {
+    IFXEmailContactCreateEdit,
+    IFXPhoneContactCreateEdit,
+    IFXFullContactCreateEdit,
+  },
+  props: {
+    contactType: {
+      required: false,
+      default: null
+    },
+    id: {
+      required: false,
+      default: null
+    }
+  },
   data() {
     return {
       allUsers: [],
+      localContactType: this.contactType,
+      item: null
     }
   },
   methods: {
     ...mapActions(['showMessage']),
-    async init() {
-      this.item = await this.getItem()
-      this.cachedItem = JSON.parse(JSON.stringify(this.item))
-      // TODO: only get chunks
-      this.allUsers = await this.$api.user.getList()
-        .catch(err => this.showMessage(err))
-    },
     updateContactUsers(users) {
       this.$set(this.item.users, ...users)
     },
     updateContactOrganizations(organizations) {
       this.$set(this.item.organizations, ...organizations)
     },
+  },
+  computed: {
+    title() {
+      const createOrEdit = this.id ? 'Edit' : 'Create'
+      const type = this.localContactType ? `${this.localContactType} Contact` : ' Contact'
+      const id = this.item && this.item.name ? `for ${this.item.name}` : this.id || ''
```

Comment:
We should start allowing [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) so we can do things like:
`this.item?.name`